packages/debian/PKGBUILD

```diff
@@ -0,0 +1,56 @@
+# Maintainer: dada513 <dada513@protonmail.com>
+
+pkgname=polymc
+pkgver=1.0.5
+pkgrel=1
+pkgdesc="Minecraft launcher with ability to manage multiple instances."
+arch=('i686' 'x86_64')
+url="https://github.com/PolyMC/PolyMC"
+license=('GPL3')
+depends=('libqt5core5a' 'libqt5network5' 'libqt5gui5')
+conflicts=('multimc' 'polymc')
+makedepends=('cmake' 'git' 'openjdk-17-jdk' 'zlib1g-dev' 'libgl1-mesa-dev')
+optdepends=('java-runtime=8: support for Minecraft versions < 1.17'
+            'java-runtime=17: support for Minecraft versions >= 1.17')
+source=("git+https://github.com/PolyMC/PolyMC"
+        "git+https://github.com/MultiMC/libnbtplusplus"
+        "git+https://github.com/PolyMC/quazip")
+
+sha256sums=('SKIP'
+            'SKIP'
+            'SKIP')
+
+prepare() {
+  cd "${srcdir}/PolyMC"
+  git checkout ${pkgver}
```

Comment:
You should be able to replace this with the following under your `source` array:

```sh
source=("git+https://github.com/PolyMC/PolyMC#tag=${pkgver}")
```